roaringbitmap/src/main/java/org/roaringbitmap/RoaringBitmap.java

```diff
@@ -1429,6 +1430,24 @@ public boolean contains(long minimum, long supremum) {
   }
 
 
+  /**
+   * Deserialize (retrieve) this bitmap.
+   * See format specification at https://github.com/RoaringBitmap/RoaringFormatSpec
+   *
+   * The current bitmap is overwritten.
+   *
+   * @param in the DataInput stream
+   * @param buffer a byte[] used to buffer DataInput reading
```

Comment:
"The buffer gets overwritten with data during deserialization. You can pass a NULL value as a buffer. A buffer containing at least 8192 bytes might be ideal for performance. It is recommended to reuse the buffer between calls to deserialize (in a single-threaded context) for best performance."

Reply:
That is, here I think we should provide guidance as to what "buffer" is and how it is supposed to be used.

Reply:
Done